values/strings.xml

```diff
@@ -757,7 +749,7 @@
     <string name="settings_winter">Winter</string>
     <string name="notify_riot_reason">There\'s a riot because of %1$s!</string>
     <string name="notify_riot_noreason">There\'s a riot for no reason!</string>
-    <string name="notify_demonstration">There\'s a demonstration in our town because of %1s! Call the police or inprove happiness, or something bad will happen!</string>
+    <string name="notify_demonstration">There\'s a demonstration in your city because of %1s! Call the police or improve happiness, or a riot will happen!</string>
```

Comment:
Should be kept our instead of your, since the notifier is a worker in this town.